### PR TITLE
[new] Knowledge Base manage backups

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Moved the [article](https://kb-managed.timescale.com/en/articles/2940373-managing-your-backups) to docs repos

# Links

Fixes #[insert issue link, if any]

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

### CODE REVIEW

**Review by Loquacity — COMMENTED** [review]
Haven't finished reviewing, sorry, Rajie, I'll come back to it!

**Comment by Rajakavitha1** [comment]
> Haven't finished reviewing, sorry, Rajie, I'll come back to it!

no worries!!!!

File: mst/manage-backups.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+For information about installing and using `timescaledb-backup`, see [installing timescaledb-backup][github-timescaledb].
+
+## 'pg_dump'
+The `pg_dump` command allows you to create backups that can be directly restored elsewhere if need be. Typical parameters for the command include:
+
+```bash
+pg_dump '<SERVICE_URL_FROM_PORTAL>' -f '<TARGET_FILE/DIR>' -j '<NUMBER_OF_JOBS>' -F '<BACKUP_FORMAT>'
+```
+The `pg_dump` command can be run also against one of the standby nodes.
```
Comment (charislam) [inline mst/manage-backups.md:35]:
```suggestion

The `pg_dump` command can also be run against one of the standby nodes. 
```

File: mst/manage-backups.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+```bash
+pg_dump '<SERVICE_URL_FROM_PORTAL>' -f '<TARGET_FILE/DIR>' -j '<NUMBER_OF_JOBS>' -F '<BACKUP_FORMAT>'
+```
+The `pg_dump` command can be run also against one of the standby nodes. 
+For example, to create a backup in directory format which can be used directly with `pg_restore` using two concurrent jobs and storing the results to a directory named `backup`:
+```bash
+pg_dump 'postgres://tsdbadmin:password@mypg-myproject.a.timescaledb.io:26882/defaultdb?sslmode=require' -f backup -j 2 -F directory
+```
+You can put all backup files to single tar file and upload to S3:
```
Comment (charislam) [inline mst/manage-backups.md:40]:
```suggestion

You can put all backup files to single tar file and upload to S3:

```

File: mst/manage-backups.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+For information about installing and using `timescaledb-backup`, see [installing timescaledb-backup][github-timescaledb].
+
+## 'pg_dump'
+The `pg_dump` command allows you to create backups that can be directly restored elsewhere if need be. Typical parameters for the command include:
+
+```bash
+pg_dump '<SERVICE_URL_FROM_PORTAL>' -f '<TARGET_FILE/DIR>' -j '<NUMBER_OF_JOBS>' -F '<BACKUP_FORMAT>'
+```
+The `pg_dump` command can be run also against one of the standby nodes. 
+For example, to create a backup in directory format which can be used directly with `pg_restore` using two concurrent jobs and storing the results to a directory named `backup`:
```
Comment (charislam) [inline mst/manage-backups.md:36]:
```suggestion
For example, use the following command to create a backup in directory format using two concurrent jobs. The results are stored to a directory named `backup`:

```

File: mst/manage-backups.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+```bash
+pg_dump 'postgres://tsdbadmin:password@mypg-myproject.a.timescaledb.io:26882/defaultdb?sslmode=require' -f backup -j 2 -F directory
+```
+You can put all backup files to single tar file and upload to S3:
+```bash
+export BACKUP_NAME=backup-date -I.tartar -cf $BACKUP_NAME backup/s3cmd put $BACKUP_NAME s3://pg-backups/$BACKUP_NAME
+```
+[github-timescaledb]: https://github.com/timescale/timescaledb-backup/#installing-timescaledb-backup
```
Comment (charislam) [inline mst/manage-backups.md:44]:
```suggestion

[github-timescaledb]: https://github.com/timescale/timescaledb-backup/#installing-timescaledb-backup
```

**Review by charislam — APPROVED** [review]

**Comment by charislam** [comment]
oh wait I think this is still missing a page-index update!

File: mst/manage-backups.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+excerpt: Understand the various alternative tools that you can use to manage your MST backups
+product: mst
+keywords: [backup, manage, restore, tiemscaledb-backup, pg_dump]
+tags: [backup, manage, restore, timescaledb-backup, pg_dump]
+---
+
+# Manage your backups
+Managed Service for TimescaleDB provides a point-in-time recovery (PITR). To restore service from a backup, click the `Restore` button in the `Backups` tab for your service. The backups are taken automatically by Managed Service for TimescaleDB and retained for a few days depending on your plan type. You can also alternatives, such as `timescaledb-backup` and `pg_dump` to create a backup and restore the TimescaleDB.
+
+## 'timescaledb-backup'
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply mst/manage-backups.md:11]: Thanks @charislam !!! I was not sure that is why I added it. I will delete it now.

**Comment by Rajakavitha1** [comment]
> oh wait I think this is still missing a page-index update!

just updated!!! thank you!!!

**Review by Loquacity — APPROVED** [review]

File: mst/manage-backups.md [hunk 6]
```diff
@@ -1,44 +1,35 @@
...
 ```
+
 You can put all backup files to single tar file and upload to S3:
 ```bash
 export BACKUP_NAME=backup-date -I.tartar -cf $BACKUP_NAME backup/s3cmd put $BACKUP_NAME s3://pg-backups/$BACKUP_NAME
 ```
-[github-timescaledb]: https://github.com/timescale/timescaledb-backup/#installing-timescaledb-backup
\ No newline at end of file
+
+[github-timescaledb]: https://github.com/timescale/timescaledb-backup/#installing-timescaledb-backup
```
Comment (Loquacity) [inline mst/manage-backups.md:35]: Do you need to add this reference back in?

Comment (Rajakavitha1) [reply mst/manage-backups.md:35]: sorry!!!! it was added inadvertently!!! thanks for spotting

**Review by alexthinath — APPROVED** [review]

**Pull request merged**